src/components/settings/SettingsRow.tsx

```diff
@@ -42,7 +43,7 @@ const SettingsRowComponent = (props: Props) => {
   const [pending, handlePress] = usePendingPress(onPress)
 
   return (
-    <TouchableHighlight accessible={false} underlayColor={theme.settingsRowPressed} style={styles.row} onPress={handlePress}>
+    <EdgeTouchableHighlight accessible={false} underlayColor={theme.settingsRowPressed} style={styles.row} onPress={handlePress}>
```

Comment:
This guy has also incorporated the `usePendingPress` hook.

Reply:
Has animation logic tied to `pending`